src/modules/core/components/MembersList/MembersListItem.tsx

```diff
@@ -67,9 +86,28 @@ const MembersListItem = <U extends AnyUser = AnyUser>(props: Props<U>) => {
         // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
         tabIndex={onRowClick ? 0 : undefined}
       >
+        {userReputationData && tokenInfoData && (
+          <div className={styles.reputationSection}>
+            <Icon
```

Comment:
Lol! This is so nitpicky...

But... In the specs, the star icon is after the reputation value:
![Screenshot from 2020-11-25 14-06-24](https://user-images.githubusercontent.com/1193222/100225863-85cd8200-2f27-11eb-9f08-9eae735fa0a0.png)

While yours is before:
![Screenshot from 2020-11-25 14-06-29](https://user-images.githubusercontent.com/1193222/100225872-89f99f80-2f27-11eb-9793-c68eda10af47.png)

Please reverse it :)